### PR TITLE
SS14-26480 Roll Traversal Distorter into regular Artifact Analyzers

### PR DESCRIPTION
**Pull request by FairlySadPanda**
## About the PR
Bit of a grab-bag of cleanup of Xenoarchaeology prompted by #26480.

## Why / Balance
Xenoarch needed a bit of cleanup but a lot of the work I thought it needed was already either done or already was present before writing 26480. This covers the remaining mald points.

## Technical details
1. Traversal distortion biases are now "up" and "down" instead of "in" and "out".
2. Node generation has been tidied up to make it a little clearer how it works.
3. Traversal Distorters have been removed from the game along with their board.
4. Traversal distortion is now done by the artifact analyzers by default.
5. Some loc strings have been made clearer.
6. The Abnormal Artifact Manipulation tech has been made slightly cheaper.
7. The aformentioned tech is now localized as Artifact Recycling, given all it does is unlock the crusher.
8. The Xenoarchaeology guidebook entry has been given a bit of a rewrite, putting all information into one page and making sure to cover the basics that otherwise would require someone typing "Liltenhead artifact tutorial" into Youtube.

## Media
- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**
:cl:
- tweak: Xenoarchaeology Traversal Distorters have been rolled into the Artifact Analyzers to make artifact gameplay a bit more controlled.
- tweak: The T2 Abnormal Artifact Manipulation tech has been made cheaper to compensate for losing an unlock.
- tweak: The Xenoarchaeology guidebook entry has been rewritten to be more useful and comprehensive for newbies.

### CODE REVIEW

**Comment by FairlySadPanda** [comment]
Note that the guidebook not mentioning the crusher is a conscious choice. The guidebook should mostly be for common roundstart questions - if the RD or someone unlocks the tech and builds the crusher, they should need to explain to their newbies what it does and how to make use of it.

**Comment by EmoGarbage404** [comment]
modifying the traversal distorter state should be done from the analysis console UI now that it's built into the analyzer.

**Comment by FairlySadPanda** [comment]
> modifying the traversal distorter state should be done from the analysis console UI now that it's built into the analyzer.

I can agree but that is UI work I don't want to tackle in this PR (as I understand zilch of the UI code)

**Review by lzk228 — CHANGES_REQUESTED** [review]
dont forget to put all removed entities in migration.yml

**Comment by FairlySadPanda** [comment]
https://github.com/space-wizards/space-station-14/assets/732532/e1513abc-427f-440b-843d-a2d99539eb1f

I did the thing

**Comment by Tayrtahn** [comment]
The tooltip on that button is greatly appreciated!

**Comment by FairlySadPanda** [comment]
> The tooltip on that button is greatly appreciated!

Yeah, this is already a huge buff to Xenoarch speed but players need to know how to use it

File: Content.Server/Xenoarchaeology/Equipment/Components/TraversalDistorterComponent.cs [hunk 1]
```diff
@@ -8,14 +8,14 @@
...
     public TimeSpan NextActivation = default!;
     public TimeSpan ActivationDelay = TimeSpan.FromSeconds(1);
 }
 
 public enum BiasDirection : byte
 {
-    In, //down the tree, towards depth 0
-    Out //up the tree, away from depth 0
+    Up, //down the tree, towards depth 0
+    Down, //up the tree, away from depth 0
```
Comment (0x6273) [inline Content.Server/Xenoarchaeology/Equipment/Components/TraversalDistorterComponent.cs:20]: `Up` is down the tree and `Down` is up the tree??

**Review by 0x6273 — CHANGES_REQUESTED** [review]

File: Content.Server/Xenoarchaeology/Equipment/Components/TraversalDistorterComponent.cs (hunk 1 above)
Comment (FairlySadPanda) [reply Content.Server/Xenoarchaeology/Equipment/Components/TraversalDistorterComponent.cs:20]: Good catch, the comments are wrong :) Fixing!

**Comment by FairlySadPanda** [comment]
I'm going to ignore conflcts to migration.yml as I'm vaguely sure any maintainer hitting the big green merge button can handle resolving the conflict 

or pinging me to resolve the conflict

**Comment by lzk228** [comment]
idk but it feels kinda weird with those colors, and maybe you can put the extract button to the bottom, because its like very important button
image

File: Resources/ServerInfo/Guidebook/Science/Xenoarchaeology.xml [hunk 2]
```diff
@@ -22,13 +32,52 @@ While it might seem random to an untrained eye, a skilled scientist can learn to
...
+
+The traversal distorter used to be a dedicated machine, but has recently been built into artifact analyzers. It aids in moving through the nodes of an artifact.
+<Box>
+  <GuideEntityEmbed Entity="MachineArtifactAnalyzer"/>
+</Box>
+Artifacts placed on top of a powered artifact analyzer are subjected to a bias which affects which node they will move to after being activated. The bias can be set by clicking on the analyzer.
+
+There are two types of biases:
+- [color=#a4885c]Up:[/color] favors nodes closer to the origin. Results in a decrease of depth.
+- [color=#a4885c]Down:[/color] favors nodes farther away from the origin. Results in an increase of depth.
```
Comment (EmoGarbage404) [inline Resources/ServerInfo/Guidebook/Science/Xenoarchaeology.xml:58]: you can basically just trim this down to the final 2 bullet points and stick it in the rest of the article. no need to reference a nonexistent machine.

**Review by EmoGarbage404 — CHANGES_REQUESTED** [review]

File: Resources/ServerInfo/Guidebook/Science/Xenoarchaeology.xml [hunk 3]
```diff
@@ -2,18 +2,28 @@
 # Xenoarchaeology
 Xenoarchaeology is a science subdepartment focused on researching and experimenting on alien artifacts.
 
-## Artifacts
+At the start of each shift, the Science department will have access to (hopefully) at least two artifacts to experiment on. You can buy more by talking to the Cargo department.
```
[the remark this answers is not in the document]
Comment (FairlySadPanda) [reply Resources/ServerInfo/Guidebook/Science/Xenoarchaeology.xml:5]: "Will have access to at least two" is wrong as some maps are not mapped correctly. Ergo I've changed it to "usually will have access to two".

File: Resources/ServerInfo/Guidebook/Science/Xenoarchaeology.xml (hunk 2 above)
Comment (FairlySadPanda) [reply Resources/ServerInfo/Guidebook/Science/Xenoarchaeology.xml:58]: Have shortened it. Talking about bias in its own heading makes sense.

**Review by EmoGarbage404 — APPROVED** [review]

**Pull request merged**